code/workspaces/web-app/src/api/graphqlClient.js

```diff
@@ -1,8 +1,8 @@
 import { get } from 'lodash';
 import request from '../auth/secureRequest';
-import { extendSubdomain } from '../core/getDomainInfo';
+import { createUrlFromPath } from '../core/getDomainInfo';
 
-const apiURL = `${extendSubdomain('api', 8000)}/api`;
+const apiURL = createUrlFromPath('api', '8000');
```

Comment:
Why does the `8000` need to be a string rather than an integer?

Reply:
Doesn't really need to be, will fix.